frontend/src/app/application-forms/fields/noncommercial-fields.component.spec.ts

```diff
@@ -34,27 +34,33 @@ describe('noncommercial fields', () => {
     })
   );
 
-  it('should be valid if participants is greater than 0', () => {
+  it('should be invalid if paticipants is greater than 0 and spectators is greater than 0 but a sum of less than 75', () => {
     component.noncommercialFields.controls['numberParticipants'].setValue(1);
-    expect(component.noncommercialFields.controls['numberParticipants'].valid).toBeTruthy();
+    component.noncommercialFields.controls['numberSpectators'].setValue(1);
+    component.noncommercialFields.controls['activityDescription'].setValue('test');
+    component.noncommercialFields.controls['locationDescription'].setValue('test');
+    expect(component.noncommercialFields.valid).toBeFalsy();
   });
 
-  it('should be invalid if participants is less than 1', () => {
-    component.noncommercialFields.controls['numberParticipants'].setValue(0);
-    expect(component.noncommercialFields.controls['numberParticipants'].valid).toBeFalsy();
+  it('should be valid if paticipants is greater than 0 and spectators is greater than 0 with a sum greater than 75', () => {
+    component.noncommercialFields.controls['numberParticipants'].setValue(30);
+    component.noncommercialFields.controls['numberSpectators'].setValue(45);
+    component.noncommercialFields.controls['activityDescription'].setValue('test');
+    component.noncommercialFields.controls['locationDescription'].setValue('test');
+    expect(component.noncommercialFields.valid).toBeTruthy();
   });
 
-  it('should be invalid if paticipants is 0 and spectators is greater than 0', () => {
```

Comment:
Can you say why you removed this test? I think the assertion would still be the case.